whipper/common/encode.py

```diff
@@ -65,6 +67,8 @@ def _flac_encode(self):
 # I only made it a task for now because that it's easier to integrate in
 # program/cdparanoia.py - where whipper currently does the tagging.
 # We should just move the tagging to a more sensible place.
+
+
```

Comment:
I still think the comments should move "into" the class they're commenting on, rather than being separated further from it. Does `flake8` also complain if the comments are inside the `class`?